Chat/Controllers/ContosoHandshakeController.cs

```diff
@@ -44,6 +44,18 @@ public async Task<ContosoChatTokenModel> GenerateAdhocUser()
 			return await InternalGenerateAdhocUser();
 		}
 
+		/// <summary>
+		/// Gets a refreshed token for the client
+		/// </summary>
+		/// <returns></returns>
+		[Route("refreshToken/{userMri}")]
+		[HttpGet]
+		public async Task<string> RefreshTokenAsync(string userMri)
+		{
+			var (identity, token, expiresIn) = await _userTokenManager.RefreshTokenAsync(_resourceConnectionString, userMri);
+			return token;
```

Comment:
Why are we not returning all three properties on a json object? Then the client has all the info.

Reply:
yep I am just returning back the CommunicationTokenResponse now